tachyon/math/base/semigroups.h

```diff
@@ -215,6 +215,22 @@ class MultiplicativeSemigroup {
     return ret;
   }
 
+  constexpr static std::pmr::vector<MulResult> GetSuccessivePowersPmr(
```

Comment:
Can this be moved to [the commit two after](https://github.com/kroma-network/tachyon/pull/486/commits/cda7427f921e31fb8c4144a784579f2a1a00c830) in order, since the commit is about this replacement anyway?

Reply:
It won't compile in the next commit in that case.